src/middlewares/__tests__/validator.js

```diff
@@ -0,0 +1,130 @@
+const middy = require('../../middy')
+const jsonBodyParser = require('../jsonBodyParser')
+const validator = require('../validator')
+
+describe('📦  Middleware Validator', () => {
+  test('It should validate an incoming object', () => {
+    const handler = middy((event, context, cb) => {
+      cb(null, event.body) // propagates the body as a response
+    })
+
+    const schema = {
+      required: ['headers', 'body'],
```

Comment:
@lmammino in reality this is just an example to test the strictness of the validation. I could have put in a completely arbitrary schema like `const schema = { properties: { foo: { type: 'date'}}}`. It does reflect the use that will be made of the `event`. `inputSchema` is completely up to the developer and the use case at hand.